t/27-websockets.t

```diff
@@ -62,137 +66,141 @@ subtest 'Authentication' => sub {
-    my $buf;
-    redirect_output(\$buf);
-
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message("");
-    is @{$fake_tx->{out}}[0], "1003,Received unexpected data from worker, forcing close",
-      "WS Server returns 1003 when message is not a hash";
-    ok $mock_get_worker_called, 'mocked _get_worker method has been called';
-
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => "status", jobid => 1, data => "mydata"});
-    is @{$fake_tx->{w}->{status}}[0], "mydata", "status got updated";
-
-    $buf = undef;
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => "FOOBAR"});
-    like $buf, qr/Received unknown message type "FOOBAR" from worker/, "log_error on unknown message";
-
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => 'worker_status'});
-    like($buf, qr/Could not send the population number to worker/, 'worker population unavailable')
-      or diag explain $buf;
-
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => 'worker_status'});
-    like($buf, qr/Failed updating worker seen and error status/, 'seen/error not available')
-      or diag explain $buf;
-
-    my $mock_foo = Test::MockModule->new('FooBarWorker');
-    my $mock_version_called;
-    $mock_foo->mock(websocket_api_version => sub { $mock_version_called++; undef });
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => 'worker_status'});
-    like($buf, qr/Received a message from an incompatible worker/, 'worker incompatible')
-      or diag explain $buf;
-    is @{$fake_tx->{out}}[1],
-      "1008,Connection terminated from WebSocket server - incompatible communication protocol version";
-    ok $mock_version_called, 'mocked websocket_api_version method has been called';
-
-    $buf                 = undef;
-    $mock_version_called = undef;
-    $mock_foo->mock(websocket_api_version => sub { $mock_version_called++; 0 });
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => 'property_change'});
-    like $buf, qr/Received a message from an incompatible worker/ or diag explain $buf;
-    is @{$fake_tx->{out}}[2],
-      "1008,Connection terminated from WebSocket server - incompatible communication protocol version";
-    ok $mock_version_called, 'mocked websocket_api_version method has been called';
-
-    $buf                 = undef;
-    $mock_version_called = undef;
-    $mock_foo->mock(websocket_api_version => sub { $mock_version_called++; WEBSOCKET_API_VERSION + 1 });
-    $fake_tx->OpenQA::WebSockets::Controller::Worker::_message({type => 'accepted'});
-    like $buf, qr/Received a message from an incompatible worker/ or diag explain $buf;
-    is @{$fake_tx->{out}}[3],
-      "1008,Connection terminated from WebSocket server - incompatible communication protocol version";
-    ok $mock_version_called, 'mocked websocket_api_version method has been called';
+subtest 'web socket message handling' => sub {
```

Comment:
WebSocket. 😉

Reply:
Hunspell which is the ultimate source of truth when it comes to spelling disagrees :-)  
(likely because it just knows webs and sockets)

I can change it but then I would need so many occurrences for consistency.